.gitignore

```diff
@@ -20,3 +20,4 @@
 .Trashes
 ehthumbs.db
 Thumbs.db
+.terraform.lock.hcl
```

Comment:
Looking at https://www.terraform.io/docs/language/dependency-lock.html it seems this file should be commited:
> Terraform has made some changes to the provider dependency selections recorded
in the .terraform.lock.hcl file. Review those changes and commit them to your
version control system if they represent changes you intended to make.

Reply:
Yes for terraform code where us use this module. We are here within the module - we define the dependencies in [version.tf]